LeaderboardBackend/Models/Leaderboard.cs

```diff
@@ -1,17 +1,11 @@
 using System.ComponentModel.DataAnnotations;
 
-namespace LeaderboardBackend.Models
-{
-	public class Leaderboard
-	{
-		public long Id { get; set; }
-
-		[Required]
-		public string? Name { get; set; }
+namespace LeaderboardBackend.Models;
 
-		[Required]
-		public string? Slug { get; set; }
-
-		public string? Rules { get; set; }
-	}
+public record Leaderboard
```

Comment:
Unfortunately this is not a good idea with Entity Framework Core models. https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/types/records#value-equality

Reply:
Addressed in 76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d (`Leaderboard.cs` [`R5`](https://github.com/leaderboardsgg/leaderboard-backend-poc/commit/76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d#diff-bedf8c542bf892d8011d2896ece68b4d97b443e88f6939acc1ba4e8353b5e0a4R5)).